.env.example

```
@@ -0,0 +1,32 @@
+UID=1000
+GID=1000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GID key should go before the UID key

(UnorderedKey)